packages/clay-multi-select/src/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -568,14 +568,21 @@ exports[`Interactions add non-sequential text should filter the items when it is
               </div>
             </button>
           </li>
+          <li>
+            <button
+              class="dropdown-item"
+            >
+              bar
+            </button>
+          </li>
         </ul>
       </div>
     </div>
   </div>
 </body>
 `;
 
-exports[`Interactions adding text doesn not filter the items with a custom function 1`] = `
+exports[`Interactions adding text doesn't not filter the items with a custom function 1`] = `
```

Comment:
@bryceosterhaus perhaps tests like this may no longer make sense since `sourceItems` no longer filters. What do you think?